core/src/main/res/values-ar/strings.xml

```diff
@@ -135,6 +135,12 @@
   <string name="exporting_label">جار التصدير ...</string>
   <string name="export_error_label">حدث خطأ أثناء التصدير</string>
   <string name="export_success_title">تم التصدير بنجاح</string>
+
+  <!--Favorites import and export-->
+  <string name="favorites">المفضلات</string>
+  <string name="favorites_export_label">تصدير المفضلات</string>
+  <string name="favorites_export_summary">تصدير المفضلات الى ملف</string>
```

Comment:
Translations are only done on Transifex. Merging changes from GitHub and Transifex is messy, so please remove those. You can later translate on Transifex after merging.

Reply:
I missed that part in `CONTRIBUTING.md`. Sorry, I'll remove and add them on Transifex.